modules/dynamic-plugins/proc-using-topology-plugin.adoc

```diff
@@ -2,19 +2,21 @@
 Topology is a front-end plugin that enables you to view the workloads as nodes that power any service on the Kubernetes cluster.
 
 .Prerequisites
-* You have installed the Red Hat Developer Hub (RHDH).
+* You have installed the {product} ({product-very-short}).
 * You have installed the Topology plugin. 
 //For the installation process, see Installation.
-* If the RBAC permission framework is enabled, ensure to add the following permission policies in an external permission policies configuration file named `rbac-policy.csv` to allow the RBAC admins or your desired user(s)/group(s) to access the Topology plugin:
+* If the RBAC permission framework is enabled, add the following permission policies in an external permission policies configuration file `rbac-policy.csv` to allow the RBAC admins or your desired user(s)/group(s) to access the Topology plugin:
```

Comment:
Tense mismatch: present tense (add) denotes an action the user has to do, therefore a procedure step. In the prerequisites we expect past tense (added).